specs/latest/1.0/index.html

```diff
@@ -453,13 +453,13 @@ <h2 class="no-toc">Table of contents</h2>
         textures, buffers (i.e., VBOs), framebuffers, renderbuffers, shaders and
         programs. The WebGLRenderingContext interface has a method to create a
         WebGLObject subclass for each type. Data from the underlying graphics library are
-        stored in these objects and are fully managed by them. The resources represented
-        by these objects are guaranteed to exist as long as the object exists.
-        Furthermore, the DOM object is guaranteed to exist as long as the author has an explicit
-        valid reference to it <b>or</b> as long as it is bound by the underlying graphics library.
-        When none of these conditions exist the user agent can, at any point, delete the object
-        using the equivalent of a delete call (e.g., deleteTexture). If authors wish to control
-        when the underlying resource is released then the delete call can be made explicitly.
+        stored in these objects and are fully managed by them.
+        The DOM object is guaranteed to exist at least as long as the author has an explicit
+        valid reference to it <b>and</b> at least as long as it is bound by the underlying graphics
+        library.
```

Comment:
This update is incorrect. For example, calling bindTexture keeps the associated WebGLTexture object alive, even if the author releases all of their references to it, because they can call for example getParameter(TEXTURE_BINDING_2D) and get the reference back. The expando-loss conformance tests verify these behaviors.

The conjunction should be "or".